templates/base.html

```diff
@@ -152,6 +153,31 @@
   }
   {% endswitch %}
 
+  var gender = '';
```

Comment:
@waseem-omar is there a reason why you: 

1. don't compute the relevant variables in the view and pass them into the context, or even just put them into the session for use in templates? Using ``cleaned_data`` is a little unusual and possibly dangerous.
1. don't just put the entirety of the JS setting into the template ``if``/``then`` statements? Why have all the extra JS conditionals and computation?